test/ts/migrations/4_deploy_terms_contracts_test.ts

```diff
@@ -0,0 +1,66 @@
+// External Libraries
+import * as chai from "chai";
+
+// Utils
+import { Web3Utils } from "../../../utils/web3_utils";
+import ChaiSetup from "../test_utils/chai_setup";
+
+// Wrappers
+import { SimpleInterestTermsContractContract as SimpleInterestTermsContract } from "../../../types/generated/simple_interest_terms_contract";
+
+import { CollateralizedSimpleInterestTermsContractContract as CollateralizedTermsContract } from "../../../types/generated/collateralized_simple_interest_terms_contract";
+
+import { IncompatibleTermsContractContract as IncompatibleTermsContract } from "../../../types/generated/incompatible_terms_contract";
+
+import { CollateralizerContract as Collateralizer } from "../../../types/generated/collateralizer";
+
+ChaiSetup.configure();
+
+const expect = chai.expect;
+const web3Utils = new Web3Utils(web3);
+
+contract("Migration #4: Deploying Terms Contracts", async (ACCOUNTS) => {
+    const CONTRACT_OWNER = ACCOUNTS[0];
+    const TX_DEFAULTS = { from: CONTRACT_OWNER, gas: 4000000 };
+
+    let simpleInterestTermsContract: SimpleInterestTermsContract;
```

Comment:
why do the contracts need to be specified in this higher scope? can they just not be referenced within the individual tests?